### PR TITLE
Day fields fetched from date and filters

### PR DESCRIPTION
**Pull request by vimalraj27**
Day fields fetched from date and filters in:

- Delivery Note
- Delivery Trip
- Sales Invoice
- Sales Order

**Screenshots:**

Delivery Note:
Screenshot from 2022-07-07 14-42-33

Sales Invoice:
Screenshot from 2022-07-07 14-43-34

Sales Order:
Screenshot from 2022-07-07 14-47-48

Delivery Trip:
Screenshot from 2022-07-07 15-47-51
Screenshot from 2022-07-07 15-48-52
Screenshot from 2022-07-07 15-49-20
Screenshot from 2022-07-07 15-51-01

### CODE REVIEW

File: sks/sks/custom/js/delivery_note.js [hunk 1]
```diff
@@ -115,6 +115,15 @@ frappe.db.get_single_value("Thirvu Retail Settings","allow_only_if_delivery_note
 		})
 	}
  })
+
+ frappe.ui.form.on("Delivery Note",{
+	onload:function(frm,cdt,cdn){
+		var a = new Date(cur_frm.doc.posting_date);
```
Comment (Gokulnath17) [inline sks/sks/custom/js/delivery_note.js:121]: Change the field name properly

File: sks/sks/custom/js/sales_invoice.js [hunk 2]
```diff
@@ -155,11 +155,20 @@ frappe.db.get_single_value("Thirvu Retail Settings","allow_only_if_sales_invoice
...
 	}
  })
- 
-
-
 
 
+ frappe.ui.form.on("Sales Invoice",{
+	onload:function(frm,cdt,cdn){
+		var a = new Date(cur_frm.doc.due_date);
```
Comment (Gokulnath17) [inline sks/sks/custom/js/sales_invoice.js:162]: Change the field name properly

File: sks/sks/custom/js/sales_invoice.js [hunk 3]
```diff
@@ -155,11 +155,20 @@ frappe.db.get_single_value("Thirvu Retail Settings","allow_only_if_sales_invoice
...
+		var weekdays=["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"]
+		cur_frm.set_value("due_day",weekdays[a.getDay()])
+	},
+	due_date:function(frm,cdt,cdn){
+		var a = new Date(cur_frm.doc.due_date);
```
Comment (Gokulnath17) [inline sks/sks/custom/js/sales_invoice.js:167]: Change the field name properly

File: sks/sks/custom/js/sales_order.js [hunk 4]
```diff
@@ -205,6 +205,19 @@ frappe.ui.form.on("Sales Order",{
     }
 })
 
+frappe.ui.form.on("Sales Order",{
+	delivery_date:function(frm,cdt,cdn){
+		var a = new Date(cur_frm.doc.delivery_date);
```
Comment (Gokulnath17) [inline sks/sks/custom/js/sales_order.js:210]: Change the field name properly

File: sks/sks/custom/js/sales_order.js [hunk 5]
```diff
@@ -205,6 +205,19 @@ frappe.ui.form.on("Sales Order",{
...
+		var weekdays=["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"]
+		cur_frm.set_value("delivery_day",weekdays[a.getDay()])
+	},
+    onload:function(frm,cdt,cdn){
+		var a = new Date(cur_frm.doc.delivery_date);
```
Comment (Gokulnath17) [inline sks/sks/custom/js/sales_order.js:215]: Change the field name properly

**Review by Gokulnath17 — CHANGES_REQUESTED** [review]
Change the requests

File: sks/sks/custom/js/delivery_note.js (hunk 1 above)
Comment (vimalraj27) [reply sks/sks/custom/js/delivery_note.js]: Requestes changes done

File: sks/sks/custom/js/sales_invoice.js (hunk 2 above)
Comment (vimalraj27) [reply sks/sks/custom/js/sales_invoice.js]: Requestes changes done

File: sks/sks/custom/js/sales_invoice.js (hunk 3 above)
Comment (vimalraj27) [reply sks/sks/custom/js/sales_invoice.js]: Requestes changes done

File: sks/sks/custom/js/sales_order.js (hunk 4 above)
Comment (vimalraj27) [reply sks/sks/custom/js/sales_order.js]: Requestes changes done

File: sks/sks/custom/js/sales_order.js (hunk 5 above)
Comment (vimalraj27) [reply sks/sks/custom/js/sales_order.js]: Requestes changes done

**Pull request merged**